src/Enum/TxSendErrors.ts

```diff
@@ -1,8 +0,0 @@
-export enum TxSendErrors {
```

Comment:
There are good error enum, but the descriptions are not really descriptive.

Reply:
This file is removed, can you please clarify what you're referring to? :thinking: